.prow.yaml

```diff
@@ -97,7 +96,7 @@ presubmits:
           privileged: true
         resources:
           requests:
-            cpu: 3000m
+            cpu: 5000m
```

Comment:
why don't you just use `5` instead of this miliCPUs. Forgot why we haven't done it in the first place, probably we haven't thought about it.